### ISSUE TITLE
Push only once

### ISSUE THREAD

**Issue by wvffle** [dtinth/todo-actions#65]
Can we push only once? It kind of spams my update channel, when there are 3 messages about the pushes.

**Comment by dtinth** [comment]
@wvffle 

There's a lightweight version of this project here which is a bit different but pushes once https://github.com/dtinth/todo-actions-lite

btw, I'm currently not maintaining this project, please feel free to fork it.

**Comment by wvffle** [comment]
> btw, I'm currently not maintaining this project, please feel free to fork it.

Yeah, I have a fork with a bunch of fixes. The issue there was a TODO for me to implement in the future ;p
It still requires some work, but when I'm finished, #61 could be merged :)

**Comment by wvffle** [comment]
> There's a lightweight version of this project

I like more the behavior of this project though ;)
